src/components/InviteUserModal.tsx

```
@@ -0,0 +1,200 @@
+        onClose()
+        onSuccess?.()
+      } else {
+        toast({
+          title: 'Erro',
+          description: result.error?.userMessage || 'Erro ao convidar usuário',
+          status: 'error',
+          duration: 5000,
+          isClosable: true,
+        })
+      }
+    } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)